py/picca/delta_extraction/utils.py

```diff
@@ -82,8 +82,10 @@
     "LY10": 919.3514,
 }
 
-ACCEPTED_BLINDING_STRATEGIES = ["none", "minimal", "corr_yshift"]
-
+ACCEPTED_BLINDING_STRATEGIES = [
+    "none", "minimal", "corr_yshift", "dasi_m2", "desi_y1", "desi_y3"]
```

Comment:
Should we remove corr_yshift from this list?

Reply:
can do, but then we won't be able to run with old deltas (see the other thread)